python/cudf_polars/cudf_polars/dsl/expressions/datetime.py

```diff
@@ -114,6 +114,7 @@ def __init__(
         self.options = options
         self.name = name
         self.children = children
+        self.is_pointwise = all(c.is_pointwise for c in self.children)
```

Comment:
@wence- - I will remove this implicit traversal in favor of an explicit traversal over the expression graph in `Select`. However, before I make the necessary changes to `traversal`, let me know if you like this better (the logic in `Select` is obviously simpler, but the `Expr` classes must check their own children)

Reply:
I noted above, I prefer not to traverse here.

Reply:
>I noted above, I prefer not to traverse here.

Okay, yeah. This is what I expected. Was just personally easier for me to think through the problem the other way (sorry), but I am planning to move the traversal back to Select.